packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -91,9 +91,7 @@ describeFullCompat("Named root data stores", (getTestObjectProvider) => {
 	 * Gets an aliased data store with the given id. Throws an error if the data store cannot be retrieved.
 	 */
 	async function getAliasedDataStoreEntryPoint(dataObject: ITestFluidObject, id: string) {
-		// back-compat: Older container runtime did not have getAliasedDataStoreEntryPoint.
-		const dataStore = await (runtimeOf(dataObject).getAliasedDataStoreEntryPoint?.(id) ??
-			runtimeOf(dataObject).getRootDataStore(id, false /* wait */));
+		const dataStore = await runtimeOf(dataObject).getAliasedDataStoreEntryPoint(id);
 		if (dataStore === undefined) {
```

Comment:
I know you said this PR was simple, and it is, but you could make it more complex by updating this helper to just return `undefined` here (or remove the helper), and then you can remove the try-catch blocks in the tests.  Up to you

Reply:
I already removed the try-catch blocks in the previous PR. I didn't return undefined here but threw an error on undefined and the call site does assert.doesNotReject. Otherwise, I would had to update all asser.ok calls in the test.

Reply:
There was one remaining which I removed here.